transbank-sdk-dotnet-core-rest-example/Controllers/TransaccionCompleta/MallFullTransactionDeferredController.cs

```diff
@@ -80,7 +80,7 @@ public ActionResult Commit(String token, String child_commerce_code, String chil
                 null,
                 gracePeriod
             ));
-
+            token ??= TempData["Token"] as string;
```

Comment:
Can token be non-null here? If it has a token, shouldn't it be left unchanged?

Reply:
Code changes were made to omit this validation.